rholang/src/test/scala/coop/rchain/rholang/interpreter/package.scala

```diff
@@ -12,14 +12,10 @@ import coop.rchain.rholang.interpreter.accounting._
 
 package object test {
 
-  def interpreter[M[_]: Sync](): M[Interpreter[M]] =
-    for {
-      costAlg <- CostAccounting.of[M](Cost.Max)
-      cost    = loggingCost[M](costAlg, noOpCostLog)
-      result = {
-        //implicit val c = cost
-        Interpreter[M]
-      }
-    } yield result
+  def interpreter[M[_]: Sync](): Interpreter[M] = {
```

Comment:
I'm curious why you opted for the impure version of this function? All of the uses I saw are within a monadic context. Most are something like `interpreter[Task].evaluate(runtime, fileContents.mkString)`, which can easily be made `interpreter[Task].map(_.evaluate(runtime, fileContents.mkString))`. If you decide to revert to the pure version, you should remove the call parenthesis because they'll falsely indicate impurity.

Reply:
This is no longer needed